public/app/plugins/datasource/loki/querybuilder/operations.ts

```diff
@@ -420,3 +420,19 @@ export function getOperationDefinitions(): QueryBuilderOperationDef[] {
 
   return list;
 }
+
+// Keeping a local copy as an optimization measure.
+const definitions = getOperationDefinitions();
+
+/**
+ * Given an operator, return the corresponding explain.
+ * Conceibed for usage within the Query Editor.
+ */
+export function explainOperator(id: LokiOperationId | string): string {
```

Comment:
I am wondering if this should be `explainOperation` or `explainCompletionItem` as it explains different operations and not just operators.

Reply:
The whole thing is called `operations`, so I followed that pattern.

Reply:
Not sure I understand what you mean here.  In my opinion, `explainOperator` should be `explainOperation` as it explains any operations, not just operators such as `=`, `!=`,....

Reply:
Ohhhh, gotcha, I misread your comment. `explainOperation` is a good suggestion. I'll update it in one of my open PRs. Thanks!